data/headers/windows/String.h

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
same comment as before, let's add some license comments here. Other than that, looks good

Reply:
Sorry, I forgot!